src/handlers/createHandler.ts

```diff
@@ -122,6 +123,16 @@ type InternalEventHandlers = {
   onGestureHandlerStateChange?: (event: any) => void;
 };
 
+let showedRngh2Notice = false;
+function showRngh2NoticeIfNeeded() {
+  if (!showedRngh2Notice && isDevEnvironment()) {
```

Comment:
checking `isDevEnvironment` won't be necessary here if you apply the below comment